packages/2d/src/components/CodeBlock.ts

```diff
@@ -47,14 +50,44 @@ export interface CodeModification {
 }
 
 export class CodeBlock extends Shape {
-  private static initialized = createComputedAsync(
-    () => ready().then(() => true),
-    false,
-  );
-
   @initial('tsx')
   @signal()
-  public declare readonly language: SimpleSignal<string, this>;
+  public declare readonly language: SimpleSignal<Lang, this>;
+
+  @initial(undefined)
+  @signal()
+  public declare readonly stockTheme: SimpleSignal<Theme | undefined, this>;
```

Comment:
The default value should be `null` to be consistent with other Node properties:
```suggestion
  @initial(null)
  @signal()
  public declare readonly stockTheme: SimpleSignal<Theme | null, this>;
```